### ISSUE TITLE
Overviewer terminates when requested dimension isn't found

### ISSUE THREAD

**Issue by computertechie** [overviewer/Minecraft-Overviewer#1202]
If a dimension for a render in the config file is not found, Overviewer will terminate, rather than skipping that render and continuing with the rest of the defined renders.

The following was used to attempt to render a newly created world (ie, only the overworld exists, no nether or End).

Config file: https://gist.github.com/computertechie/b822ee7898db28eba826
Output:

````
2015-02-05 11:24:34  Welcome to Minecraft Overviewer!
2015-02-05 11:24:34  Generating textures...
2015-02-05 11:24:39  Generating textures...
2015-02-05 11:24:45  Generating textures...
2015-02-05 11:24:50  Generating textures...
*******************************************************************************
2015-02-05 11:24:55 E Sorry, you requested dimension 'nether' for survival_nether, but I couldn't find it
````

**Comment by Gizmokid2005** [comment]
Fixed in PR #1203.

**Comment by CounterPillow** [comment]
Closed as it is a duplicate of #913
